logviewer/api/proxy.go

```diff
@@ -7,18 +7,27 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/Netflix/titus-executor/utils/netns"
-
+	"github.com/Netflix/titus-executor/logviewer/conf"
 	"github.com/Netflix/titus-executor/utils"
+	"github.com/Netflix/titus-executor/utils/netns"
+	"github.com/Netflix/titus-executor/vpc/tool/cni"
 )
 
 // The URL to connect to once we've done a Setns() into the container
 const inContainerURL = "http://127.0.0.1:8004"
 
 func getNetNsPath(containerID string) string {
+	if conf.KubeletMode {
+		return cni.NSAliasPath(containerID)
```

Comment:
If we make a symlink here, just so we can use that alternate path in KubeleMode, why don't we just use the actual file in the first place instead of having an alternate mode and the symlink?

Reply:
The symlink affords a "wellknown" name that we can devine from just a pod.  The actual filename contains an unrelated guid that I could not lookup without querying containerd which would mean I would have to query kubernetes first to get the container id to ask about.  The symlink seemed to be a fast way to add a weak reference by pod id, but open to other ideas for mapping between the two.

Reply:
Gotcha. I assumed that guid was actually the task name, but I guess that doesn't make much sense now that I know it is containerd doing it.